pkg/repository/manager.go

```diff
@@ -276,37 +265,20 @@ func (m *Manager) EnsureFork(ctx context.Context, repo *Repository) error {
 	return err
 }
 
-// used to help mocking os.Rename
-// TODO(hilalymh): Q4 switch to go1.16 os/fs library/interface
-var renameDirectory = os.Rename
-
 func (m *Manager) EnsureClone(ctx context.Context, repo *Repository) error {
 	err := m.clone(ctx, repo.Name)
 	if err != nil && err != ErrRepositoryAlreadyExist {
 		return err
 	}
 
-	// At this point we ensured that the fork repository is cloned. We need to rename it
-	// if there is any fork prefix.
-	if repo.Name != repo.ExpectedForkName {
-
-		newPath := filepath.Join(
-			filepath.Dir(repo.FullPath),
-			repo.Name,
-		)
-		err := renameDirectory(repo.FullPath, newPath)
-		if err != nil {
-			return err
-		}
-	}
```

Comment:
why delete this block of code? wasn't it working as expected?

Reply:
I just cloned directly into `repo.FullPath` instead. There was no need to clone it with the fork as the name and then rename it.